gtk4/src/subclass/editable.rs

```diff
@@ -21,12 +21,12 @@ pub trait EditableImpl: WidgetImpl {
         self.parent_changed(editable)
     }
 
-    fn get_text(&self, editable: &Self::Type) -> GString {
-        self.parent_get_text(editable)
+    fn text(&self, editable: &Self::Type) -> GString {
+        self.parent_text(editable)
     }
 
-    fn get_delegate(&self, editable: &Self::Type) -> Option<Editable> {
-        self.parent_get_delegate(editable)
+    fn delegate(&self, editable: &Self::Type) -> Option<Editable> {
```

Comment:
One of the issues with not using the migration tools ([fn def](https://crates.io/crates/fix-getters-def) and [call sites](https://crates.io/crates/fix-getters-calls)) is that you don't take advantage of the heuristics for deciding whether a function is a getter for sure.

This function is not a getter in the sense: "a function that retrieves current value of a field". After the rename, it reads more like it performs an imperative `delegate` action. I think we lost a meaning in the process. There are many examples like this in the manual updates. You also take the risk of missing some [useful substitutions](https://github.com/fengalin/fix-getters/blob/26c950680699cdb8a096ff454bd2f858c018a0a7/rules/src/function.rs#L81) or not renaming things like `get_mut_structure` as the more common `get_structure_mut` among other cases I identified when migrating `gtk-rs`.

I strongly advise to follow this workflow:

- Regen with https://github.com/gtk-rs/gir/pull/1021.
- Check the result and update `Gir.toml` if needed.
- Run the migrations tools (you can install them using `cargo`):
  - `fix-getters-def`
  - `fix-getters-calls`
  - `rust fmt`
- Compile and fix the remaining candidates on a case by case basis.

An additional benefit for this is that application developers will be able to use the same heuristics (using the [call sites](https://crates.io/crates/fix-getters-calls) migration tool) to ease their application migration process. Not using these is like throwing my work away :). The point in these tools was that `sed` was not enough to do the migration properly.

I can do the work on `gtk4-rs` if you don't have time to go through this. Just let me finish the on-going migration of `gstreamer-rs` & `gst-plugins-rs`.

Reply:
Well, the thing is it's a getter. The semantics are a bit different here because it's a virtual function but at the end, you must return the Editable implementation you want to "delegate" and at the end you do return an internal field. 

I will regenerate using your tool, but I believe this function should be renamed nonetheless.

Reply:
IMHO, it's not a substitute to using a `struct` field, in the sense of calling a function to read a field that you could read directly if the field was `pub`. In this case, you *retrieve* a value by indicating a type, that's why I believe we should keep `get` here.

Reply:
Oh, I just got your point about the sublassing part. So it might not be the best example. But I think we should get an agreement on the range for which we decide to remove the `get` prefix. I expected a debate on this, but it didn't happen.

Reply:
Makes sense, at least for this one as after looking at the source code of Editable. The `delegate` is not a property per see but just an internal thing with a getter method to read it.

Reply:
Please, tell me what you think about the general selection rules I listed there: https://github.com/fengalin/fix-getters#get-functions-selection.